js/passage.js

```diff
@@ -1,4 +1,127 @@
-const passage = Object.freeze([
+export class Passage {
+  constructor(verses) {
+    this._verses = verses;
+  }
+
+  metadata() {
+    let firstVerse = this._verses[0];
+    let lastVerse = this._verses[this._verses.length - 1]
+    let bookChapter = firstVerse.book + ' ' + firstVerse.chapter + ':';
+    if (this._verses.length > 1) {
+      if (firstVerse.chapter === lastVerse.chapter) {
```

Comment:
Definitely.  I was ignoring that case for now as we don't have multiple books and not sure if that would a case for the app, but could add a test for it.